Modules/Server/Sources/PocketCastsServer/Public/Helpers/URLRequest+RefreshIfNeeded.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+
+public extension URLRequest {
+
+    mutating func setRefreshHeadersUsing(cachedResponse: CachedURLResponse) {
```

Comment:
Maybe rename it to `setEtagAndLastModifiedHeaders` to be explicit about what it does?

Reply:
Ok

Reply:
Done here: https://github.com/Automattic/pocket-casts-ios/pull/1955/commits/1cffb0c49b6567f3d7b266d6c767bd281b406b87